### ISSUE TITLE
ChillerHeater:Absorption:DirectFired IDD Mistakes

### ISSUE THREAD

**Issue by asparke2** [NREL/EnergyPlus#5379]
Found in version 8.4:

The documentation for several of the curve fields for the ChillerHeater:Absorption:DirectFired say that they accept Biquadratic curves.  The example file `gasAbsorptionChillerHeater.idf` shows Biquadratic curves in these fields, and the `Temperature Curve Input Variable` field defines what to use for the second independent variable.  However, the IDD does not allow Biquadratic curves:

```
  A8   ,  \field Cooling Capacity Function of Temperature Curve Name
            \note The CoolCapFT curve represents the fraction of the cooling capacity of the chiller as it
            \note varies by temperature.  The curve is normalized so that at design conditions the
            \note value of the curve should be 1.0.  This is a biquadratic curve with the
            \note input variables being the leaving chilled water temperature and either
            \note the entering or leaving condenser water temperature.
            \type object-list
            \object-list LinearCurves
            \object-list QuadraticCurves
            \object-list CubicCurves
            \object-list QuarticCurves
            \object-list ExponentCurves
            \object-list UniVariateTables
```

**Comment by asparke2** [comment]
@kbenne FYI This mistake is also present in the OS version of the E+ IDD.

**Comment by mjwitte** [comment]
Those snuck in way back in v7.2.  ChillerHeater:Absorption:DoubleEffect also has the same issue.

**Comment by mjwitte** [comment]
@Myoldmopar Would you like a quick PR to fix these?

**Comment by Myoldmopar** [comment]
Door is still open due to the VRF issue and some abrupt dentistry... go ahead and open a PR and if it is easily reviewed by @JasonGlazer or someone, merge it in.

**Comment by Myoldmopar** [comment]
Fixed via #5380
